docusaurus.config.js

```diff
@@ -34,6 +34,11 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      metadata: [
+        { name: 'apple-mobile-web-app-title', content: 'PGM Documentation' },
+        { property: 'og:site_name', content: 'PGM Documentation' },
+        { name: 'theme-color', content: '#4C0704' }
```

Comment:
I've noticed that `#4C0704` is similar to the colour for the OCN/OCC shield, which we retrofitted to the PGM shield. What are your thoughts on using the more neutral orange colours `#FFA500` ("orange" in CSS), or `#FF4500`? If a dark colour is more nicer, we should use `#d93b00` or `#b33000` which follows the colour theme.